runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPlugin.java

```diff
@@ -85,6 +118,10 @@
 	static final DecimalFormat POINTS_FORMAT = new DecimalFormat("#,###");
 	private static final String SPLIT_REGEX = "\\s*,\\s*";
 	private static final Pattern ROTATION_REGEX = Pattern.compile("\\[(.*?)]");
+	private static final String IMGUR_CLIENT_ID = "30d71e5f6860809";
```

Comment:
If this is doing the same thing as the screenshot plugin, it should be broken out into a util .

Reply:
Screen capture code moved out from ScreenshotPlugin to ScreenCapture util